### DISCUSSION TITLE
TLS certificate is being provisioned (stuck!!!)

### DISCUSSION THREAD

**Discussion by doncabreraphone** [community/community#140606] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi! 

I have am stuck at **"TLS certificate is being provisioned. This may take up to 15 minutes to complete."**

The domain A records are OK. Everything is OK. We actually had SSL through our Github Pages and a custom domain but a team member pushed a change to the repo and it is now not provisioning the SSL.

I've checked every discussion here and it seems my TLS cert might be "stuck". Can someone help? We have a lunch this week!

Thanks!

repo: **https://github.com/La-Reverie/lareverie.studio**
**custom domain:** https://lareverie.studio

**Comment by Ankitjarwall** [comment]
hello @doncabreraphone 
i think you are having issue with GitHub pages, you are trying to use GitHub pages with custom domain and now you are not able use ssl/tls.
you want your domain to be secure with https. Right ? i Hope this is your question if yes.

Steps to get ssl on custom domain via GitHub page :
Step 1 : make the build of your react project, currently you have uploaded the raw code to the repo (To build the react project use "npm run build") and once your build file are done upload those file to the repo in root folder, root folder should contain the index.html file.

Step 2: now go to settings > pages > branch (by default root in most cases).
         wait for the website to publish on GitHub pages, once it is done (you will get a link something like this " https://lareverie.github.io/la-Reverie.github.io ")
         
Step 3 : connect you domain, you have to add the A record , TXT record to your domain provider. once done, click on verify domain in github > page settings, make sure you have checked the https checkbox button. (At this point you may see the same issue you are still not seeing https in your website, this happens but to solve this, unpublish your website and publish it  again, this would probably solve your issue.)

if you still face any issue feel free to communicate.

Hope you find your solution after this.

Thank you

**Comment by ebndev** [comment]
Hi @doncabreraphone, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `Pages` category, as this topic may be more relevant there and might help someone give you a nudge in the right direction. 😁

In the meantime, please feel encouraged to explore existing discussions in the category and utilize the search in case other users may have run into the same issue!

**Reply by doncabreraphone** [reply]
I've search other discussions and I've seen some people with this problem, some have been solved by GitHub admins, others are unsolved.

**Reply by Sanidhyafeaturist** [reply, marked as answer]
It appears that your TLS certificate for the custom domain "lareverie.studio" is stuck in the provisioning process after a recent change to your GitHub Pages repository. Ensure that your A records are correctly configured and that no conflicting DNS settings exist. If the issue persists beyond the usual provisioning time, consider contacting GitHub support for assistance, as they may need to manually resolve the certificate provisioning.

**Comment by hannahwauchope** [comment]
Hi @doncabreraphone - did you ever get this fixed? I'm having the same issue and not sure what's up. I've done all the right things re checking A records etc
